cli/training.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import os
+import requests
+from azure.storage.blob import BlockBlobService, ContentSettings
+from operations import (
+    read_config,
+    CONFIG_PATH,
+    get_azure_storage_client
+)
+
+class TagData(object):
```

Comment:
@samiyaakhtar This should probably not live in the CLI area. Eventually we should move to a shared location.

Reply:
Okay, I can make the refactor happen as part of this PR.